spec/controllers/admin/booths_controller_spec.rb

```diff
@@ -77,6 +77,14 @@
         it 'shows success message' do
           expect(flash[:notice]).to match('Booth successfully created.')
         end
+
+        it 'creates a new user' do
```

Comment:
how is this related? what are you testing here?

Reply:
@Ana06 yes sorry i will explain it more.